mutiny-core/src/nodemanager.rs

```diff
@@ -1356,6 +1358,37 @@ impl<S: MutinyStorage> NodeManager<S> {
             return Ok(());
         }
 
+        // Skip syncing RGS if we are in safe mode.
+        if self.safe_mode {
+            log_info!(self.logger, "Skipping rgs sync in safe mode");
+        } else {
+            let last_rgs_sync_timestamp = self
+                .gossip_sync
+                .network_graph()
+                .get_last_rapid_gossip_sync_timestamp();
+
+            if let Some(rgs_url) =
+                get_rgs_url(self.network, &self.user_rgs_url, last_rgs_sync_timestamp)
+            {
+                log_info!(self.logger, "RGS URL: {rgs_url}");
+
+                let now = utils::now().as_secs();
+                let fetch_result = fetch_updated_gossip(
+                    rgs_url,
+                    now,
+                    last_rgs_sync_timestamp.unwrap_or_default(),
+                    &self.gossip_sync,
+                    &self.storage,
+                    &self.logger,
+                )
+                .await;
+
+                if fetch_result.is_err() {
+                    log_warn!(self.logger, "Failed to fetch updated gossip data");
+                }
+            }
```

Comment:
I don't think we should be fetching gossip every minute. This works well in the background but it should just run once.